apps/src/netsim/netsim.js

```diff
@@ -1083,6 +1097,19 @@ NetSim.prototype.onShardChange_= function (shard, localNode) {
   this.visualization_.setShard(shard);
   this.visualization_.setLocalNode(localNode);
   this.render();
+
+  // TODO (bbuchanan): Tear this out when replacing reset option with real UI.
+  if (shard && this.shouldResetShard() && confirm("Are you sure?" +
+          "  This will kick everyone out and reset all data for the class.")) {
+    shard.resetEverything(function (err) {
+      if (err) {
+        logger.error(err);
+        return;
+      }
+      // Reload page without the shard-reset query parameter
+      location.href=location.href.replace(/&?resetShard([^&]$|[^&]*)/i, "");
```

Comment:
again, even though this is temporary, it still feels like we're at a pretty high risk for making unintentional modifications